src/content/en/2019/performance.md

```diff
@@ -3,14 +3,14 @@ part_number: II
 chapter_number: 7
 title: Performance
 authors: [rviscomi]
-reviewers: [JMPerez, obto, sergeychernyshev, zeman]
+reviewers: [JMPerez,obto,sergeychernyshev,zeman]
 ---
 
 ## Intro
 
 Performance is a visceral part of the user experience. For [many websites](https://wpostats.com/), an improvement to the user experience by speeding up the page load time aligns with an improvement to conversion rates. Conversely, when performance is poor, users don't convert as often and have even been observed [rage clicking](https://blog.fullstory.com/rage-clicks-turn-analytics-into-actionable-insights/) on the page in frustration.
 
-There are many ways to quantify web performance. The most important thing is to measure what actually matters to users. Events like `onload` or `DOM content loaded` may not necessarily reflect what users experience visually. For example, an email client might have a very fast onload event but the only thing loaded is the interstitial progress bar, meanwhile the inbox contents are loading asynchronously. The loading metric that matters for this web app is the "time to inbox", and focusing on the `onload` event may be misleading. For that reason this chapter will look at more modern and universally applicable paint, load, and interactivity metrics to try to capture how users are actually experiencing the page.
+There are many ways to quantify web performance. The most important thing is to measure what actually matters to users. However, events like `onload` or `DOMContentLoaded` may not necessarily reflect what users experience visually. For example, an email client might have a very fast `onload` time but the only thing loaded is the interstitial progress bar, meanwhile the inbox contents are loading asynchronously. The loading metric that matters for this web app is the "time to inbox", and focusing on the `onload` event may be misleading. For that reason this chapter will look at more modern and universally applicable paint, load, and interactivity metrics to try to capture how users are actually experiencing the page.
```

Comment:
> the only thing loaded is the interstitial progress bar, meanwhile the inbox contents are loading asynchronously

Can you make it a bit more clear that the inbox is currently empty and has not finished loading yet?

Reply:
Rewrote this part. PTAL.